lib/src/components/NxPageHeader/__tests__/NxPageHeader.test.tsx

```diff
@@ -5,84 +5,119 @@
-    expect(component.children()).toMatchElement(props.children);
+    const homeLink = screen.getByRole('link', {name: 'Home'}),
+      homeLinkLogo = homeLink.querySelector('img');
+
+    expect(homeLink).toBeInTheDocument();
+    expect(homeLinkLogo).toBeInTheDocument();
   });
 
-  it('passes a logo with a webpack-generated path and proper alt tag', function() {
-    const logo = shallow(getShallowComponent().prop('logo'));
+  it('renders links in header when given links prop', function() {
+    renderEl({links: [{ name: 'foo', href: '#bar' }, { name: 'baz', href: '#qux' }]});
+
+    const link1 = screen.getByRole('link', {name: 'foo'}),
+      link2 = screen.getByRole('link', {name: 'baz'})
 
-    // this path comes from imgMock.ts
-    expect(logo).toHaveDisplayName('img');
-    expect(logo).toHaveProp('src', 'path/to/asset.png');
-    expect(logo).toHaveProp('alt', 'Sonatype');
-    expect(logo).toHaveClassName('nx-product__logo-image');
+    expect(link1).toBeInTheDocument();
+    expect(link2).toBeInTheDocument();
   });
 
-  it('includes the productInfo.name in the productInfoContent .nx-product__name if provided', function() {
+  it('renders the product name if provided', function() {
     const props = {
           productInfo: {
             name: 'test app'
           }
         },
-        defaultProductInfoContent = getShallowComponent().prop('productInfoContent'),
-        richProductInfoContent = shallow(getShallowComponent(props).prop('productInfoContent'));
+        header = renderEl(),
+        headerWithProductName = renderEl(props),
+        headerWithProductNameText = headerWithProductName?.querySelector('div.nx-product__name');
 
-    expect(defaultProductInfoContent).toBe(null);
-    expect(richProductInfoContent).toContainMatchingElement('.nx-product__name');
-    expect(richProductInfoContent.find('.nx-product__name')).toHaveText('test app');
+    expect(header?.querySelector('div.nx-product__name')).not.toBeInTheDocument();
+    expect(headerWithProductNameText).toBeInTheDocument();
+    expect(headerWithProductNameText?.textContent).toEqual('test app');
+    
   });
 
-  it('includes the productInfo.version in the productInfoContent if provided', function() {
-    const noVersionProps = {
+  it('renders the product version if provided', function() {
+    const props = {
           productInfo: {
             name: 'test app'
           }
         },
-        props = {
+        propsWithVersion = {
           productInfo: {
             name: 'test app',
             version: '1.2.3'
           }
         },
-        noVersionProductInfoContent = shallow(getShallowComponent(noVersionProps).prop('productInfoContent')),
-        richProductInfoContent = shallow(getShallowComponent(props).prop('productInfoContent'));
+        elNoProductVersion = renderEl(props),
+        elWithProductVersion = renderEl(propsWithVersion),
+        productVersion = renderEl(elWithProductVersion?.querySelector('.nx-product__version')!);
```

Comment:
Also an impl detail.  Just check that the header overall contains the version text

Reply:
> Also an impl detail. Just check that the header overall contains the version text

Done: [15b9325](https://github.com/sonatype/sonatype-react-shared-components/pull/935/commits/15b9325c914cc126a4aa0df665457670492f84c9)